strings/strings.yaml

```diff
@@ -3,17 +3,29 @@
 # the disadvantage is that this needs a custom loader and so it won't work in mocha
 welcome:
   output: >-
-    Welcome to sensors are down, a space combat game.
+    Welcome to Sensors are Down, a space combat game.
     <audio src="https://s3.amazonaws.com/sensorsaredown-static-files/mp3/explosion.mp3" />
-    Would you like to play? Say yes to start the game, no to quit, or how do I play to learn how to play.
+  firstTimePrompt: It looks like you're new. Would you like to learn how to play the game?
+  firstTimeReprompt: Say yes to learn how to play, no to start the game without instructions, or quit to quit.
```

Comment:
I love this, but you should actually add a quit intent to the speechAssets folder and to the list of intents in the FIRSTSTARTMODE.